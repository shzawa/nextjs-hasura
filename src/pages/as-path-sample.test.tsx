import { render } from '@testing-library/react';
import AsPathSample from './as-path-sample';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      pathname: '/',
      query: {},
      asPath: '/',
      basePath: '/',
      isLocaleDomain: true,
      isReady: true,
      push: jest.fn(),
      prefetch: jest.fn(),
      replace: jest.fn(),
      reload: jest.fn(),
      back: jest.fn(),
      beforePopState: jest.fn(),
      events: {
        on: jest.fn(),
        off: jest.fn(),
        emit: jest.fn(),
      },
      isFallback: false,
      isPreview: false,
    };
  },
}));

describe('初期表示', () => {
  test('サンプルページ', () => {
    const { asFragment } = render(<AsPathSample />);
    expect(asFragment()).toMatchSnapshot();
  });
});
