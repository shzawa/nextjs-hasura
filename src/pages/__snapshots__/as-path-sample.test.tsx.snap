// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`初期表示 サンプルページ 1`] = `
<DocumentFragment>
  <h1>
    このページの path は / です。
  </h1>
</DocumentFragment>
`;
